src/Toolbar/components/FontSizeButton.tsx

```diff
@@ -29,39 +29,39 @@ export const FontSizeButton = (props: FontSizeButtonProps) => {
       {/* Popover */}
       {showPopover && !readOnly && (
         <div className="rt-headerPopover">
-          <button
+          <div
```

Comment:
How do screen readers know that this is now a button?

Reply:
Good point! I'll see if I can move the popover outside of the button and still position it properly